tomopy/prep/alignment.py

```diff
@@ -69,7 +77,11 @@
            'add_jitter',
            'add_noise',
            'blur_edges',
-           'shift_images']
+           'shift_images',
+           'find_slits_corners_aps_1id',

```

Comment:
I'd try to find a way to drop aps_1id parts from these functions and provide a generic description to them. After-all these can be named as standard image processing functions. More comments/suggestions to follow when I check what they do.

Reply:
If this is the case, I think it is fine. I was trying to learn of anyone else would need these functions. An alternative would be to use the technique as a descriptor rather than the beamline, because beamlines can change in time.

Reply:
If the current setup here at 1-ID had a dedicated name, I would most likely use that instead of the beamline ID.  Unfortunately, I am not aware of whether this kind of setup has its own name, which leaves us with the beamline ID as the only option for now.